### PR TITLE
Updating various READMEs to try and better call out benefits, drawbacks and the target audience

### PR DESCRIPTION
**Pull request by tyler-ball**
Also removing old, out of date cruft

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,52 @@
...
+Chef Provisioning is maintained according to the [Chef Maintenance Policy](https://github.com/chef/chef-rfc/blob/master/rfc030-maintenance-policy.md).
+
+Drivers
+-------------
+
+Due to the age and engagement level with different Drivers they have fallen into two categories - maintained and unmaintained.
+
+Maintained drivers have a representative in the maintenance policy and get triaged / discussed during office hours. They are
+
+[AWS](https://github.com/chef/chef-provisioning-aws) | [Azure](https://github.com/chef/chef-provisioning-azure) | [Fog](https://github.com/chef/chef-provisioning-fog)
```
Comment (stuartpreston) [inline README.md]: I would suggest the Azure (classic) driver be demoted to unmaintained status. The ```machine``` resource when used with this driver is extremely limited, and the whole project relies on a fork of the Azure SDK that I created.  I would also propose we untangle these drivers from the ChefDK.

File: README.md [hunk 2]
```diff
@@ -1,17 +1,52 @@
...
+---- | ---- | ----
+[![Gem Version](https://badge.fury.io/rb/chef-provisioning-aws.svg)](http://badge.fury.io/rb/chef-provisioning-aws) | [![Gem Version](https://badge.fury.io/rb/chef-provisioning-azure.svg)](http://badge.fury.io/rb/chef-provisioning-azure) | [![Gem Version](https://badge.fury.io/rb/chef-provisioning-fog.svg)](http://badge.fury.io/rb/chef-provisioning-fog)
+
+One driver is not maintained by Chef:
```
Comment (stuartpreston) [inline README.md]: There is also the driver for [Azure Resource Manager](https://github.com/pendrica/chef-provisioning-azurerm) - this is more up to date with regards to dependencies on the Azure SDK, but also maintained separately.  Unless you want me to move it over to the Chef org (which I am happy to do!)

**Comment by stuartpreston** [comment]
+1 from me this is a better position than before and makes clear the levels of support to be expected.

**Comment by keen99** [comment]
👍 
definitely helps clarify the current state of things, thanks!

**Comment by thehar** [comment]
+1

File: README.md (hunk 1 above)
Comment (tyler-ball) [inline README.md]: https://github.com/chef/chef-provisioning-azure/pull/51

File: README.md (hunk 2 above)
Comment (tyler-ball) [inline README.md]: Added AzureRM!

**Pull request merged**